### DISCUSSION TITLE
Scalability of ntex for High-Traffic Applications

### DISCUSSION THREAD

**Discussion by mmhdi** [ntex-rs/ntex#386] [Q&A] [answered]
Dear ntex Team,

I am a software engineer at a major bank with a platform that handles Millions of requests daily. We are considering using ntex for our high-traffic website and client dashboard.

Can ntex effectively handle such a large volume of traffic? Any insights or recommendations would be greatly appreciated.

Thank you.

**Comment by fafhrd91** [comment, marked as answer]
even if you handle 100m reqs per day, it is just 1.2k req/second. at this load ntex wont be bottleneck in any form.

**Reply by mmhdi** [reply]
I know, but I wanted to make sure it's sound in all aspects for intensive production use. Thank you.

**Reply by fafhrd91** [reply]
it is designed with performance in mind. i use it in high-traffic service. in any case you would need to worry about ntex performance if you get tens of thousands reqs per sec per core.
